package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "license": "Apache License 2.0",
   "config": {
-    "webservice_version": "1.9.0-alpha.5"
+    "webservice_version": "feature/organizations-orcid"
```

Comment:
From the build log, this won't work (only actual artifacts uploaded to artifacts.oicr.on.ca will) 
i.e. need a release from webservice develop to point at